chromium_src/chrome/installer/setup/setup_main.cc

```diff
@@ -44,5 +105,23 @@ void SavePromoCode(installer::InstallStatus install_status) {
 #define DoLegacyCleanups         \
   SavePromoCode(install_status); \
   DoLegacyCleanups
+
+// The macros BRAVE_HANDLE_NON_INSTALL_CMD_LINE_OPTIONS, UpdateInstallStatus and
+// BRAVE_UPDATE_INSTALL_STATUS contain code that used to be upstream and had to
+// be restored in Brave to support delta updates on Windows until we are on
+// Omaha 4. See github.com/brave/brave-core/pull/31937.
+#define BRAVE_HANDLE_NON_INSTALL_CMD_LINE_OPTIONS                         \
```

Comment:
These defines are not the ones that we will extend, so having explicit names seems acceptable here. Just noting this because this naming scheme doesn't align with our patching rules where we use `BRAVE_CLASS_NAME_FUNCTION_NAME` scheme for defines.

Reply:
I wasn't aware of this convention and changed the macro names to follow it now in f26cc00aad5005fc9fa30709ffd607662d58eb99. Wdyt?